docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -228,76 +232,42 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
    b. Use `mimirtool` to update the configuration.
 
    ```bash
-   mimirtool config convert --yaml-file cortex.yaml
+   mimirtool config convert --yaml-file cortex.yaml --yaml-out mimir.yaml
+   ```
+
+   c. Clean up the generated YAML configuration.
+   You have to remove some fields that are generated by `mimirtool config convert` for the Helm chart installation, because the
+   mimir-distributed Helm chart has already set the default value for them. Use the following script to clean up
+   those fields:
+
+   ```bash
+   # require yq v4
+   yq -i 'del(.activity_tracker.filepath,.alertmanager.data_dir,.compactor.data_dir)' mimir.yaml
```

Comment:
Nice idea. Should we also mention all the token paths too? such as `ingester.ring.tokens_file_path`. And the rest of the paths that use `/data` in the chart - e.g. `blocks_storage.bucket_store.sync_dir` and `blocks_storage.tsdd.dir`?

Reply:
The purpose of yq script above is to "clean up" default value that can result to failed installation, i.e pointing writable folder to non `/data` folder. 

`.activity_tracker.filepath,.alertmanager.data_dir,.compactor.data_dir` are the fields that has "wrong" default value  which set to folder outside `/data` during my testing using minimal overrided configuration. Therefore I added these yq step to clean up that "wrong" default value on those 3 fields only. 

The additional fields that you mention has a sane default value. I.e tokens_file_path default value is `""` and all `sync_dir` folder default value in cortex helm are already pointing to `/data` sub folder.

Reply:
Yes, your explanation makes sense for the `ingester.ring.tokens_file_path` field. I will add that to the list to be cleaned up too.

Reply:
Yes, your explanation makes sense for the `ingester.ring.tokens_file_path` field. I will add that to the list to be cleaned up too.

Reply:
Added for rule-path and runtime-config too.